cache-config/t3c-request/t3c-request.go

```diff
@@ -68,4 +68,5 @@ func main() {
 			os.Exit(3)
 		}
 	}
+	defer cfg.TCCfg.TOClient.WriteFsCookie(toreq.FsCookiePath)
```

Comment:
This doesn't need to be a `defer`, since it's at the end of the function anyway

Reply:
Removed defer